### PR TITLE
EXPLAIN: add document for explain/explain analyze of mpp query

### PR DESCRIPTION
**Pull request by CharLotteiu**
### What is changed, added or deleted? (Required)

Add the document for `explain/explain analyze` of mpp query.

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-cn/pull/5754
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Comment by CharLotteiu** [comment]
Waiting on https://github.com/pingcap/docs/pull/5113 to fix the dead anchor.

File: TOC.md [hunk 1]
```diff
@@ -101,6 +101,7 @@
     + [Overview](/sql-tuning-overview.md)
     + Understanding the Query Execution Plan
       + [Overview](/explain-overview.md)
+      + [MPP Queries](/explain-mpp.md)
       + [`EXPLAIN` Walkthrough](/explain-walkthrough.md)
```
Comment (TomShawn) [inline TOC.md:105]:
```suggestion
      + [`EXPLAIN` Walkthrough](/explain-walkthrough.md)
      + [MPP Queries](/explain-mpp.md)
```

File: explain-mpp.md [hunk 2]
```diff
@@ -0,0 +1,170 @@
...
+|       ├─ExchangeReceiver_20(Build)     | 6.00    | cop[tiflash] |               |                                                |
+|       │ └─ExchangeSender_19            | 6.00    | cop[tiflash] |               | ExchangeType: Broadcast                        |
+|       │   └─Selection_18               | 6.00    | cop[tiflash] |               | not(isnull(test.t1.id))                        |
+|       │     └─TableFullScan_17         | 6.00    | cop[tiflash] | table:a       | keep order:false                               |
+|       └─Selection_22(Probe)            | 6.00    | cop[tiflash] |               | not(isnull(test.t1.id))                        |
+|         └─TableFullScan_21             | 6.00    | cop[tiflash] | table:b       | keep order:false                               |
++----------------------------------------+---------+--------------+---------------+------------------------------------------------+
+```
+
+The work process in the above execution plan is as follows:
```
Comment (TomShawn) [inline explain-mpp.md:138]: ditto. Ordered list is better for `process`.

**Review by TomShawn — COMMENTED** [review]
Very minor comments~

File: explain-mpp.md [hunk 3]
```diff
@@ -0,0 +1,170 @@
...
+|       │     └─TableFullScan_16         | 6.00    | cop[tiflash] | table:a       | keep order:false                                   |
+|       └─ExchangeReceiver_23(Probe)     | 6.00    | cop[tiflash] |               |                                                    |
+|         └─ExchangeSender_22            | 6.00    | cop[tiflash] |               | ExchangeType: HashPartition, Hash Cols: test.t1.id |
+|           └─Selection_21               | 6.00    | cop[tiflash] |               | not(isnull(test.t1.id))                            |
+|             └─TableFullScan_20         | 6.00    | cop[tiflash] | table:b       | keep order:false                                   |
++----------------------------------------+---------+--------------+---------------+----------------------------------------------------+
+12 rows in set (0.00 sec)
+```
+
+The work process in the above execution plan is as follows:
```
[the remark this answers is not in the document]
Comment (CharLotteiu) [reply explain-mpp.md:110]: Not sure whether the first two operations are executed one by one or simultaneously. @windtalker Could you please take a look?

[the remark this answers is not in the document]
Comment (CharLotteiu) [reply explain-mpp.md:110]:
```suggestion
In the above execution plan:
```

File: explain-mpp.md (hunk 2 above)
Comment (CharLotteiu) [reply explain-mpp.md:141]:
```suggestion
In the above execution plan:
```

File: TOC.md [hunk 4]
```diff
@@ -102,6 +102,7 @@
     + Understanding the Query Execution Plan
       + [Overview](/explain-overview.md)
       + [`EXPLAIN` Walkthrough](/explain-walkthrough.md)
+      + [MPP Queries](/explain-mpp.md)
       + [Indexes](/explain-indexes.md)
       + [Joins](/explain-joins.md)
       + [Subqueries](/explain-subqueries.md)
```
Comment (CharLotteiu) [inline TOC.md:108]:
```suggestion
      + [Indexes](/explain-indexes.md)
      + [Joins](/explain-joins.md)
      + [MPP Queries](/explain-mpp.md)
      + [Subqueries](/explain-subqueries.md)
```

File: TOC.md [hunk 5]
```diff
@@ -101,6 +101,7 @@
     + [Overview](/sql-tuning-overview.md)
     + Understanding the Query Execution Plan
       + [Overview](/explain-overview.md)
+      + [MPP Queries](/explain-mpp.md)
```
[the remark this answers is not in the document]
Comment (CharLotteiu) [reply TOC.md]: Updated in https://github.com/pingcap/docs/pull/5123/files#diff-19c55decc75725c7053c81221f882f7e759ced8c296087bbb5dbdb90f2d365dcR105-R108.

**Review by windtalker — COMMENTED** [review]
lgtm

**Pull request merged**